src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+// eslint-disable max-len
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  // eslint-disable-next-line
+  let [valueInput, valueFunc] = useState('');
+
+  const filterMovie = moviesFromServer.filter(movie => {
+    return movie.title.toLocaleLowerCase()
+      .indexOf(valueInput.toLocaleLowerCase()) >= 0
```

Comment:
you can move `.toLocaleLowerCase().indexOf(valueInput.toLocaleLowerCase()) >= 0` to separate function to avoid duplications
and you can use `includes()` method here instead of `indexOf()`